src/utils/async-storage.js

```diff
@@ -1,225 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
as a bit of trivia, this file originally came from Firefox OS :-) (and mostly David Flanagan)